### ISSUE TITLE
ffe-buttons: Anchors not vertically aligned with buttons in button group

### ISSUE THREAD

**Issue by wkillerud** [SpareBank1/designsystem#87]
#### Which package is this issue related to?

`ffe-buttons`

#### Describe your issue (screenshots welcome!)

![screen shot 2018-03-13 at 10 28 40](https://user-images.githubusercontent.com/1223410/37333300-5163475e-26a9-11e8-8f99-68ebd5ec8845.png)

The primary button is an `a` element in the picture above. Swapping the primary button to a `button` element and changing the secondary button to an `a` reverses the behavior:

![screen shot 2018-03-13 at 10 30 04](https://user-images.githubusercontent.com/1223410/37333394-87118b90-26a9-11e8-95e8-38ba264db945.png)

If both are `buttons` they are aligned correctly:

![screen shot 2018-03-13 at 10 30 11](https://user-images.githubusercontent.com/1223410/37333413-907afffe-26a9-11e8-8e81-794eeef5ff7a.png)

The same if both are anchors, although it seems there is some extra bottom margin. Maybe missing a reset?

![screen shot 2018-03-13 at 10 30 53](https://user-images.githubusercontent.com/1223410/37333465-b282aca0-26a9-11e8-927c-908c61e75ea8.png)

#### What is the expected behavior? 

The buttons should align vertically regardless of element type

#### What is the actual behavior?

`a` and `button` get different vertical heights.

#### Steps to reproduce

```html
<div class="ffe-button-group ffe-button-group--thin">
    <button aria-busy="false" aria-disabled="false" class="ffe-button ffe-button--primary">
        <span class="ffe-button__label">Velg minikasko</span>
        <div aria-hidden="true" aria-label="Vennligst vent" class="ffe-button__spinner"></div>
    </button>
    <a class="ffe-button ffe-button--secondary" href="/privat/forsikring/#/tilbud/bil">
        <span class="ffe-button__label">Tilpass forsikringen selv</span>
        <div aria-hidden="true" aria-label="Vennligst vent" class="ffe-button__spinner"></div>
    </a>
</div>

```

**Comment by wkillerud** [comment]
The reimplementation of the buttons made use of flexbox, so it may be some _old meets new_ thing going on here.

Adding `display: flex;` to the button group and `display: inline-flex;` to the buttons resolves the issue.

**Comment by wkillerud** [comment]
Fixed by #88
